apps/dolly-frontend/src/main/js/src/components/fagsystem/pensjon/visning/TpVisning.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from 'react'
+import SubOverskrift from '~/components/ui/subOverskrift/SubOverskrift'
+import { TitleValue } from '~/components/ui/titleValue/TitleValue'
+import Loading from '~/components/ui/loading/Loading'
+import { DollyFieldArray } from '~/components/ui/form/fieldArray/DollyFieldArray'
+import { ErrorBoundary } from '~/components/ui/appError/ErrorBoundary'
+import { Alert } from '@navikt/ds-react'
+import { fetchTpOrdninger } from '~/components/fagsystem/tjenestepensjon/form/Form'
+import { SelectOptionsManager as Options } from '~/service/SelectOptions'
+
+export const sjekkManglerPensjonData = (tpData) => {
+	return !tpData || !tpData.length
+}
+
+export const TpVisning = ({ data, loading }) => {
+	const [ordninger, setOrdninger] = useState([Options('tpOrdninger')])
+
+	fetchTpOrdninger()
+	useEffect(() => {
+		setOrdninger(Options('tpOrdninger'))
+	}, [Options('tpOrdninger')])
+
+	if (loading) return <Loading label="Laster TP forvalter-data" />
```

Comment:
Mener sonar er misfornøyd med å bruke return direkte uten {} 😅 
Samme for linjen under 🤓